config/initializers/pdp.rb

```diff
@@ -0,0 +1,2 @@
+# Initialize Atmosphere pdp class to DefaultPdp
+Atmosphere::at_pdp_class = Atmosphere::DefaultPdp
```

Comment:
This was meant to ensure backwards compatibility with legacy code which may expect to read this variable. I can remove it, but I'm not entirely sure what goes on e.g. in ISMOP etc. Perhaps safer to keep it here.